### PR TITLE
support for RHEL8 builders

### PR DESCRIPTION
**Pull request by vavuthu**
Signed-off-by: vavuthu [vavuthu@redhat.com](mailto:vavuthu@redhat.com)

### CODE REVIEW

**Comment by vavuthu** [comment]
build team job passed on RHEL 8 builders: https://url.corp.redhat.com/1bd5902

**Review by petr-balogh — DISMISSED** [review]

File: ocs_ci/utility/utils.py [hunk 1]
```diff
@@ -4794,3 +4815,17 @@ def get_glibc_version():
...
+    """
+    Gets the architecture of host
+
+    Returns:
+        str: Host architecture
+
+    """
+    cmd = "uname -m"
+    res = exec_cmd(cmd)
+    out = res.stdout.decode("utf-8")
```
Comment (dahorak) [inline ocs_ci/utility/utils.py:4830]: What about using `os.uname()` functionality here?
```
os.uname().machine
```

**Review by dahorak — APPROVED** [review]
Just suggestion to use `os.uname` instead of calling `uname` command, otherwise LGTM.

File: ocs_ci/utility/utils.py (hunk 1 above)
Comment (vavuthu) [reply ocs_ci/utility/utils.py]: done

**Review by dahorak — APPROVED** [review]

**Review by deepshikhaaa — APPROVED** [review]

**Review by petr-balogh — APPROVED** [review]

**Pull request merged**